Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -225,16 +160,14 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
         }
     }
 
-    func SELlongPressLocation(recognizer: UILongPressGestureRecognizer) {
-        if recognizer.state == UIGestureRecognizerState.Began {
-            delegate?.browserLocationViewDidLongPressLocation(self)
-        }
+    func textFieldShouldBeginEditing(textField: UITextField) -> Bool {
+        delegate?.browserLocationViewDidTapLocation(self)
```

Comment:
Is that bad? In general, I assume if the user wants to start editing, we should show the editing mode or else they can't type.